packages/localize/src/number/forceENAUSymbols.js

```diff
@@ -0,0 +1,18 @@
+export function forceENAUSymbols(formattedParts, options) {
+  const result = formattedParts;
+  const numberOfParts = result.length;
+  // Change the symbol from JPY to ¥, due to bug in Chrome
+  if (numberOfParts > 1 && options && options.currencyDisplay === 'symbol') {
+    if (options.currency === 'EUR') {
+      result[0].value = '€';
+      result[1].value = '';
+    } else if (options.currency === 'USD') {
+      result[0].value = '$';
+      result[1].value = '';
+    } else if (options.currency === 'JPY') {
+      result[0].value = '¥';
+      result[1].value = '';
+    }
```

Comment:
```suggestion
    switch(options.currency) {
      case 'EUR':
        result[0].value = '€';
        break;
      case 'USD':
        result[0].value = '$';
        break;
      case 'JPY':
        result[0].value = '¥';
        break;
        /* no default */
    }
    result[1].value= '';
```

Reply:
Hey, thanks for the suggestions, I was able to fix the nested if (with the switch) and changed the comment on the file. Kindly let me know if the fixes are okay.